### PR TITLE
Add support for evil-respect-visual-line-mode

### PR DESCRIPTION
**Pull request by mk3z**

### CODE REVIEW

File: evil-colemak-basics.el [hunk 1]
```diff
@@ -146,6 +146,17 @@ rotated; see evil-colemak-basics-rotate-t-f-j."
         "m" 'evil-backward-char)
       (evil-define-key '(normal visual) keymap
         "h" 'evil-set-marker))
+    (when evil-respect-visual-line-mode
+      (evil-define-key '(motion normal visual-line-mode) keymap
```
Comment (wbolster) [inline evil-colemak-basics.el:150]: are you sure this is correct? isn't this supposed to be `visual`?

Comment (mk3z) [reply evil-colemak-basics.el:150]: That's how it's written in the source code of evil-mode: https://github.com/emacs-evil/evil/blob/master/evil-integration.el#L478

Comment (mk3z) [reply evil-colemak-basics.el]: I did some experimenting and it seems like `visual` works better.

**Review by wbolster — APPROVED** [review]

**Comment by wbolster** [comment]
merged 🚀

thanks!

**Pull request merged**